src/utils/rate-limiters/base_db_limiter.ts

```diff
@@ -16,7 +16,7 @@ export abstract class MetaTransactionBaseDbRateLimiter {
         this._dbField = dbField;
     }
 
-    public abstract async isAllowedAsync(
+    public abstract isAllowedAsync(
```

Comment:
the change is abstract functions can't be marked async?

Reply:
yup exactly